applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -288,32 +295,34 @@ void ElasticIsotropic3D::CalculateElasticMatrix(
 void ElasticIsotropic3D::CalculatePK2Stress(
     const Vector& rStrainVector,
     Vector& rStressVector,
-    const double E,
-    const double NU
+    ConstitutiveLaw::Parameters& rValues
 )
 {
-    SizeType SizeSystem = GetStrainSize();
-    Matrix C(SizeSystem,SizeSystem);
-    CalculateElasticMatrix(C, E, NU);
+    const SizeType size_system = GetStrainSize();
```

Comment:
This implementation is really inefficient...

i would implement only the rhs withut having to allocate the lhs

Reply:
@RiccardoRossi 
Do you mean without `CalculateElasticMatrix`, C?

Reply:
yes, see my comment above

Reply:
done